recipes/simbody/all/conandata.yml

```
@@ -0,0 +1,9 @@
+sources:
+  "3.7.0":
+    url: "https://github.com/simbody/simbody/archive/refs/tags/Simbody-3.7.tar.gz"
+    sha256: "d371a92d440991400cb8e8e2473277a75307abb916e5aabc14194bea841b804a"
+
+patches:
+  "3.7.0":
+    - base_path: "source_subfolder"
+      patch_file: "patches/0001-simbody-3.7.0-cmake-fixes.patch"
```

```suggestion
    - patch_file: "patches/0001-simbody-3.7.0-cmake-fixes.patch"
```
Not needed base_path, as cmake_layout uses src folder

---

done